src/conn/conn_layered.c

```diff
@@ -48,11 +48,12 @@ __layered_get_disagg_checkpoint(WT_SESSION_IMPL *session, const char **cfg,
     if (page_log->pl_get_complete_checkpoint_ext == NULL)
         WT_ERR(ENOTSUP);
 
-    WT_ERR(page_log->pl_get_complete_checkpoint_ext(page_log, &session->iface,
-      complete_checkpoint_lsn, NULL, complete_checkpoint_timestamp, complete_checkpoint_metadata));
+    ret = page_log->pl_get_complete_checkpoint_ext(page_log, &session->iface,
```

Comment:
nitpick: I would do this in a single line.

WT_ERR_NOTFOUND_OK(page_log->pl_get_complete_checkpoint_ext(page_log, &session->iface,complete_checkpoint_lsn, NULL, complete_checkpoint_timestamp, complete_checkpoint_metadata), true);